pkg/oak_run.go

```diff
@@ -70,6 +70,9 @@ func (or OakRun) JarFile() string {
 }
 
 func (or OakRun) prepare() error {
+	if err := pathx.DeleteIfExists(or.Dir()); err != nil {
```

Comment:
without it when updating e.g 

```yaml
    #url: "https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.18%2B10/OpenJDK11U-jdk_[[.Arch]]_[[.Os]]_hotspot_11.0.18_10.[[.ArchiveExt]]"
    url: "https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u432-b06/OpenJDK8U-jdk_[[.Arch]]_[[.Os]]_hotspot_8u432b06.[[.ArchiveExt]]"
  
```

then old Java was in use because previously unpacked was still there